indico/modules/events/papers/views.py

```diff
@@ -32,7 +32,13 @@ class WPDisplayPapersBase(WPJinjaMixin, WPConferenceDefaultDisplayBase):
     template_prefix = 'events/papers/'
 
     def getJSFiles(self):
-        return WPConferenceDefaultDisplayBase.getJSFiles(self) + self._asset_env['modules_papers_js'].urls()
+        return (WPConferenceDefaultDisplayBase.getJSFiles(self) +
+                self._asset_env['modules_papers_js'].urls() +
+                self._asset_env['modules_event_management_js'].urls())
```

Comment:
I'd include papers after event management - event management is more generic than papers